### PR TITLE
Cleanup unlisted packages

### PR DESCRIPTION
**Pull request by weshaggard**
Also use a more stable sort so the order doesn't bounce around

### CODE REVIEW

File: _data/allpackages/dotnet-packages.csv [hunk 1]
```diff
@@ -1,6 +1,4 @@
 "Service","Package","Version","Hide","Notes"
-"Azure.AI.InkRecognizer.UWP.Stroke","Azure.AI.InkRecognizer.UWP.Stroke","1.0.0","",""
```
Comment (weshaggard) [inline _data/allpackages/dotnet-packages.csv]: FYI @KrzysztofCwalina

**Comment by KrzysztofCwalina** [comment]
Should we also remove Azure.DCAP.Windows? It's not even a .NET package.

**Comment by weshaggard** [comment]
>Should we also remove Azure.DCAP.Windows? It's not even a .NET package.

Right now the query pulls all packages that are owned by the azure-sdk account. So if I remove it the automation will just add it back, until we unlist it from nuget. I can hide it from this table if we want to exclude it.

File: _data/allpackages/dotnet-packages.csv [hunk 2]
```diff
@@ -1,10 +1,8 @@
...
-"Azure.AI.InkRecognizer.WPF.Stroke","Azure.AI.InkRecognizer.WPF.Stroke","1.0.2","",""
 "Azure.Batch.FileStaging","Azure.Batch.FileStaging","8.0.1","",""
 "Azure.Core","Azure.Core","1.1.0","",""
 "Azure.Data.AppConfiguration","Azure.Data.AppConfiguration","1.0.0","",""
-"Azure.DCAP.Windows","Azure.DCAP.Windows","1.2.0","",""
+"Azure.DCAP.Windows","Azure.DCAP.Windows","1.2.0","true",""
```
Comment (weshaggard) [inline _data/allpackages/dotnet-packages.csv]: @KrzysztofCwalina I've just marked this package as hidden from the table until we decide what to do with it on nuget.org

Comment (Petermarcu) [reply _data/allpackages/dotnet-packages.csv]: Can we also remove the Azure.Batch one? That one is track N-1.

Comment (weshaggard) [reply _data/allpackages/dotnet-packages.csv]: Hide in our table or unlist from nuget? I'll start by hiding it in our table.

**Pull request merged**